Signal/src/view controllers/CountryCodeViewController.m

```diff
@@ -20,17 +24,61 @@ - (void)viewDidLoad {
     [self.navigationController.navigationBar setTranslucent:NO];
     _countryCodes = [self countryCodesForSearchTerm:nil];
     self.title    = NSLocalizedString(@"COUNTRYCODE_SELECT_TITLE", @"");
+    self.searchBar.delegate = self;
 }
 
 - (NSArray *)countryCodesForSearchTerm:(NSString *)searchTerm {
     NSMutableArray *result = [NSMutableArray array];
-    for (NSString *countryCode in [PhoneNumberUtil countryCodesForSearchTerm:nil]) {
+    for (NSString *countryCode in [PhoneNumberUtil countryCodesForSearchTerm:searchTerm]) {
```

Comment:
Done. See: https://github.com/WhisperSystems/SignalServiceKit/pull/128

Note that PhoneNumberUtil resides in SSK but PhoneNumberTest resides in Signal-iOS.